src/simsopt/mhd/vmec_diagnostics.py

```diff
@@ -971,13 +979,13 @@ def vmec_fieldlines(vs, s, alpha, theta1d=None, phi1d=None, phi_center=0, plot=F
         for js in range(ns):
             theta_pest[js, :, :] = theta1d[None, :]
             phi[js, :, :] = phi_center + (theta1d[None, :] - alpha[:, None]) / iota[js]
-
+    
     def residual(theta_v, phi0, theta_p_target, jradius):
```

Comment:
Because `fzero_residuals_function` is introduced, this earlier `residual` function can be removed.